### PR TITLE
Fix Scrutiny service start failure

### PR DESCRIPTION
**Pull request by asylumexp**
## Description

It fails to start because it can't access an InfluxDB instance, so just added one.

Fixes #3275

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] I have performed a self-review of my code, adhering to established codebase patterns and conventions.

### CODE REVIEW

File: install/scrutiny-install.sh [hunk 1]
```diff
@@ -21,9 +21,24 @@ $STD apt-get install -y \
...
   make \
-  mc
+  mc \
+  lsb-base \
+  lsb-release \
+  gnupg2
 msg_ok "Installed Dependencies"
 
+msg_info "Setting up InfluxDB Repository"
+wget -q https://repos.influxdata.com/influxdata-archive_compat.key
```
Comment (tteck) [inline install/scrutiny-install.sh]:
```
wget -qO- https://repos.influxdata.com/influxdata-archive_compat.key | gpg --dearmor > /etc/apt/trusted.gpg.d/influxdata-archive_compat.gpg
```

File: install/scrutiny-install.sh [hunk 2]
```diff
@@ -21,9 +21,24 @@ $STD apt-get install -y \
...
+cat influxdata-archive_compat.key | gpg --dearmor | tee /etc/apt/trusted.gpg.d/influxdata-archive_compat.gpg > /dev/null
+sh -c 'echo "deb [signed-by=/etc/apt/trusted.gpg.d/influxdata-archive_compat.gpg] https://repos.influxdata.com/debian stable main" > /etc/apt/sources.list.d/influxdata.list'
```
Comment (tteck) [inline install/scrutiny-install.sh]:
```
echo "deb [signed-by=/etc/apt/trusted.gpg.d/influxdata-archive_compat.gpg] https://repos.influxdata.com/debian stable main" > /etc/apt/sources.list.d/influxdata.list
```

**Comment by asylumexp** [comment]
fixed those

File: install/scrutiny-install.sh [hunk 3]
```diff
@@ -21,9 +21,24 @@ $STD apt-get install -y \
...
-  mc
+  mc \
+  lsb-base \
+  lsb-release \
+  gnupg2
 msg_ok "Installed Dependencies"
 
+msg_info "Setting up InfluxDB Repository"
+wget -qO- https://repos.influxdata.com/influxdata-archive_compat.key | gpg --dearmor > /etc/apt/trusted.gpg.d/influxdata-archive_compat.gpg
+cat influxdata-archive_compat.key | gpg --dearmor | tee /etc/apt/trusted.gpg.d/influxdata-archive_compat.gpg > /dev/null
```
Comment (tteck) [inline install/scrutiny-install.sh]: delete line 32

**Comment by asylumexp** [comment]
done

also this is how it was done in influxdb-install, which is why it was like that in this pr

**Pull request merged**